changelog/bug-1596615.md

```diff
@@ -0,0 +1,4 @@
+level: major
```

Comment:
Is this a [breaking change](https://github.com/taskcluster/taskcluster/blob/master/dev-docs/best-practices/changelog.md#levels) for any of our users?

Reply:
(we're discussing this in slack.... settled on `minor` level)